tests/manage/noobaa/conftest.py

```diff
@@ -0,0 +1,109 @@
+import logging
+
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import noobaa
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def noobaa_obj():
+    """
+    Returns a NooBaa resource that's connected to the S3 endpoint
+    Returns:
+        noobaa_obj: A NooBaa resource
+
+    """
+
+    noobaa_obj = noobaa.NooBaa()
+    return noobaa_obj
+
+
+@pytest.fixture()
```

Comment:
Why do we need to delete the objects if we are the deleting buckets later?

Reply:
In order to delete a bucket, you have to empty it first :)

Reply:
thanks